frontend/src/components/form/Form.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  Form as AForm, Input, Row, Typography,
+} from 'antd';
+import styled from 'styled-components';
+
+import { Button } from '@/components/buttons';
+import { Colors } from '@/styles/colors';
+
+const StyledForm = styled(AForm)`
+  background: ${Colors.onPrimary};
+  border: 1px solid ${Colors.shadow};
+  padding: 20px;
+  border-radius: 10px;
+
+  .first-and-last-name-form {
+      display: flex;
+      justify-content: space-between;
+      gap: 30px;
+  }
+
+  .first-and-last-name-form > * {
+    width: 100%;
+  }
+
+  .first-and-last-name-form Input {
```

Comment:
`Input` z dużej litery? Na pewno jest to prawidłowe zapisanie znacznika? Nie powinno to być zapisane z małej litery? Chyba, że jest to stylowanie komponentu wewnątrz (ale tego nie znam :D), także jak spełnia swoje działanie to tylko daj mi znać 😁